components/SingleScoreTable/index.tsx

```diff
@@ -0,0 +1,381 @@
+import React, { useCallback, useMemo } from 'react';
```

Comment:
Same reasons as above index file

Reply:
Just a question, what is the purpose of all these extra single[BLANK]tables. Shouldn't we be able to just add some props to the existing tables to handle the data for a single player? Plus we just pass in props to turn off things like pagination and such. I think it may be redundant to have all of these here. Can you investigate? If you have any questions feel free to ask.

Reply:
yea, I can investigate this.